app/posts/publish-teacher-training-courses/2024-06-25-course-sorting.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Course sorting
```

Comment:
I think specify which service this is in the title - just to make it more helpful to future us